README.adoc

```diff
@@ -12,410 +12,12 @@ ifdef::env-github[]
-
-[cols="1,1,1,2", options="header"]
-|===
-| Property name
-| Type
-| Default value
-| Description
-
-| `jvmArgs`
-| List<String>
-| `null`
-| Additional arguments passed to JVM when executing `Gatling` simulations
-
-| `systemProperties`
-| Map<String, Object>
-| `null`
-| Additional systems properties passed to JVM together with caller JVM system
-properties
-
-| `simulations`
-| Closure
-| `null`
-| Simulations filter.
-https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/util/PatternFilterable.html[See Gradle docs]
-for details.
-|===
-
-=== Default tasks
-
-[cols="1,1,2", options="header"]
-|===
-| Task name
-| Type
-| Description
-
-| `gatlingClasses`
-| -
-| Compiles `Gatling` simulation and copies resources
-
-| `gatlingRun`
-| GatlingRunTask
-| Executes all `Gatling` simulations configured by extension
-
-| `gatlingRun-SimulationFQN`
-| GatlingRunTask
-| Executes single `Gatling` simulation, +
-_SimulationFQN_ should be replaced by fully qualified simulation class name.
-|===
-
-.Run all simulations
-[source, bash]
-----
-  $ gradle gatlingRun
-----
-
-.Run single simulation implemented in `com.project.simu.MySimulation` class
-[source, bash]
-----
-  $ gradle gatlingRun-com.project.simu.MySimulation
-----
-
-== Troubleshooting and known issues
-
-=== Spring Boot and Netty version
-
-https://github.com/lkishalmi/gradle-gatling-plugin/issues/53[Original issue]
-
-Caused by `io.spring.dependency-management` plugin and Spring platform BOM files.
-The dependency management plugin ensures that all declared dependencies have
-exactly the same versions as declared in BOM. Since `Spring Boot` declares own
-`Netty` version (e.g. `4.1.22.Final`) - this version is applied globally for all
-the configurations of the `Gradle` project, even if configuration does not use
-`Spring`.
-
-There are 2 ways of solving the problem, depending on the actual usage of `Netty`
-in the project.
-
- * When production code does not rely on `Netty`:
-+
-.build.gradle
-[source, groovy]
-----
-ext['netty.version'] = '4.0.51.Final'
-----
-
-This declares `Netty` version globally for all transitive dependencies in your
-project, including `Spring`.
-
- * When production code uses `Netty`:
-+
-.build.gradle
-[source, groovy]
-----
-dependencyManagement {
-    gatling {
-        dependencies {
-            dependencySet(group: 'io.netty', version: '4.0.51.Final') {
-               entry 'netty-codec-http'
-               entry 'netty-codec'
-               entry 'netty-handler'
-               entry 'netty-buffer'
-               entry 'netty-transport'
-               entry 'netty-common'
-               entry 'netty-transport-native-epoll'
-            }
-        }
-    }
-}
-----
-
-These options ensure that `4.0.51.Final` will be used only for `gatling`
-configurations, leaving other dependencies unchanged.
-
-== Release a new version
-
- * Push a tag `vX.Y.Z`, where `X.Y.Z` is the version to publish.
-
```

Comment:
Perhaps add a file for dev purpose? `DEVELOPMENT.md`? `CONTRIBUTION.md`? `HOWTO.md`?

Reply:
Separate issue. Please take care of it.